app/src/test/java/org/simple/clinic/summary/PatientSummaryViewRendererTest.kt

```diff
@@ -183,4 +185,21 @@ class PatientSummaryViewRendererTest {
     verify(ui).enableContactDoctorButton()
     verifyNoMoreInteractions(ui)
   }
+
+  @Test
+  fun `hide contact doctor button if teleconsultation is disabled`() {
+    // given
+    val model = defaultModel
+        .currentFacilityLoaded(facilityWithDiabetesManagementEnabled)
```

Comment:
Can we create a separate `facilityWithTeleconsultationEnabled` here? Otherwise we are implicitly depending on the detail that `facilityWithDiabetesManagementEnabled` has this flag unset.